omnibus-base-mainnet-andromeda.toml

```diff
@@ -1,5 +1,5 @@
 name = "synthetix-omnibus"
-version = "26"
+version = "27"
```

Comment:
Just checking if we have done testnet already?

Reply:
If not, then lets do base sepolia first

Reply:
That's right we haven't